crates/ruff_linter/src/rules/pydoclint/rules/check_docstring.rs

```diff
@@ -670,17 +670,19 @@ pub(crate) fn check_docstring(
 
     let body_entries = {
         let mut visitor = BodyVisitor::new(checker.semantic());
-        visitor.visit_body(member.body());
+        visitor.visit_body(&function_def.body);
         visitor.finish()
     };
 
     // DOC201
-    if checker.enabled(Rule::DocstringMissingReturns) && docstring_sections.returns.is_none() {
-        let extra_property_decorators = checker.settings.pydocstyle.property_decorators();
-        if !definition.is_property(extra_property_decorators, checker.semantic()) {
-            if let Some(body_return) = body_entries.returns.first() {
-                let diagnostic = Diagnostic::new(DocstringMissingReturns, body_return.range());
-                diagnostics.push(diagnostic);
+    if checker.enabled(Rule::DocstringMissingReturns) {
```

Comment:
Is this change just a stylistic preference?

Reply:
Yeah, I prefer to have the enablement checks separated out from any lint-specific checks. I find it to be a clearer separation, and it would make it easier to identify and automatically rewrite them in the future.

Reply:
Makes sense, thanks